charts/authentik/values.yaml

```diff
@@ -378,9 +378,15 @@ server:
   # -- Alternative DNS policy for authentik server pods
   dnsPolicy: ""
 
-  # -- serviceAccount to use for authentik server pods
+  # -- serviceAccount to use for authentik server pods used when serviceAccount.create is false
   serviceAccountName: ~
-
+  serviceAccount:
+    create: true
```

Comment:
This definitely shouldn't be true by default, especially since we're using authentik-remote-cluster by default.